api/main.py

```diff
@@ -166,14 +166,14 @@ def add_device(house_id):
         return jsonify({"device": None, "error": {"code": 404, "message": "No such house found"}})
     data = request.get_json()
     logging.debug("Adding device: {}".format(data))
-    device = api.device_repository.add_device(house_id=ObjectId(house_id),
-                                              room_id=None,
-                                              name=data['name'],
-                                              device_type=data['device_type'],
-                                              target=data['target'],
-                                              status=data['status'],
-                                              configuration=data['configuration'],
-                                              vendor=data['vendor'])
+    device_id = api.device_repository.add_device(house_id=ObjectId(house_id),
+                                                 room_id=None,
+                                                 user_id=None,
```

Comment:
Removing house_id and only keeping user_id is fine as well. Just make sure that it's consistent for the whole API, that you update the API documentation and notify the frontend team, so they update their API calls.